### DISCUSSION TITLE
Consider publishing to winget

### DISCUSSION THREAD

**Discussion by RagibHasin** [stacked-git/stgit#401] [Ideas]
Publishing to winget would enable easier installation and automatic updates for Windows users. Also, it is not very difficult to use CI for the job.

Thanks.

**Comment by jpgrayson** [comment]
Agreed that making StGit available via the winget community repository would be nice to have. I would love help to make this happen.

**Reply by RagibHasin** [reply]
I can try to figure out a GitHub CI recipe for it.
